src/App.js

```diff
@@ -1,9 +1,23 @@
 import React from 'react'
 import { createGlobalStyle } from 'styled-components'
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+import { Composition } from 'atomic-layout'
 import Index from './pages/artist/Index'
 import Home from './pages/home/Home'
 import Album from './pages/album/Album'
+import NavBar from './molecules/NavBar'
+import MenuMobile from './atoms/MenuMobile'
+
+const areaMobile = `
+  menuMobile
+  content
+  / auto
```

Comment:
If you remove "auto", how wide will the component be? Would that be acceptable?

Reply:
Yes. It is okay.